Assets/Game/Character/Rhea/Rhea.cs

```diff
@@ -9,6 +9,8 @@ public class Rhea : Character<Rhea, RheaState, RheaStateInput>
     private float dashDuration = 0.2f;
     [SerializeField]
     private GameObject shieldPrefab = null;
+    [SerializeField]
+    private GameObject hitbox;
```

Comment:
variable names should alternate cases: EX - hitBox

Reply:
Around the internet I see both "hit box" (hitBox in camel case) and "hitbox" (hitbox in camel case) being used. I used the latter in the ticket description, so I think it's fine to leave it as is.